ports/rp2/CMakeLists.txt

```diff
@@ -188,6 +188,20 @@ target_compile_options(${MICROPY_TARGET} PRIVATE
     -Werror
 )
 
+set_source_files_properties(
+    ${PICO_SDK_PATH}/src/rp2_common/pico_double/double_math.c
+    ${PICO_SDK_PATH}/src/rp2_common/pico_float/float_math.c
+    PROPERTIES
+    COMPILE_OPTIONS "-Wno-error=uninitialized"
+)
```

Comment:
Probably related to https://github.com/raspberrypi/pico-sdk/issues/501 ?

Reply:
yes it's the same problem (I use Arch Linux as well)

Reply:
so this will just be a work-around for now, to hopefully be removed in the future